mycroft/util/json_helper.py

```diff
@@ -32,6 +33,28 @@ def merge_dict(base, delta):
             base[k] = dv
 
 
+def delete_key_from_dict(key, dictionary):
+    """Recursivily find nested key in a dict and delete it.
+
+    Arguments:
+        key (str): a period separated list of nested keys to remove
+                   eg "nested.dict.keys"
+        dictionary (dict): the dictionary to remove keys from
+    Returns:
+        Dict: original dictionary with specified keys deleted.
+    """
+    modified_dict = copy(dictionary)
+    key_list = key.split('.')
```

Comment:
should the separator be made into an argument ?

Reply:
This is true, but then we need a way to configure that delimiter so we'd be adding another mycroft.conf attribute. 

Should we just use the right data structure to begin with and make it a list of lists of strings...
```python
[['nested', 'key', 'example'], ['second', 'key']]
```

This seems really painful to write out if you want to actually use it but who knows how often it will actually get used and only has to be done once

Reply:
see https://github.com/OpenVoiceOS/ovos_utils/pull/66